source/openapi-admin-v3.yaml

```diff
@@ -7290,6 +7439,9 @@ components:
 
             - the relationship may point to many foreign documents.
             - the local field must be defined as an array in the collection schema.
+    SchemaDefinition:
+      type: object
+      description: A valid [schema](https://www.mongodb.com/docs/realm/schemas) for the collection.
```

Comment:
I can't comment on it, but I noticed that ln 7295/7447 below has a typo. "Metadata the" should probably be "Metadata that".